parm/wflow/coldstart.yaml

```diff
@@ -91,7 +91,8 @@ metatask_run_ensemble:
       SLASH_ENSMEM_SUBDIR: '&SLASH_ENSMEM_SUBDIR;'
     join: !cycstr '&LOGDIR;/{{ jobname }}_@Y@m@d@H&LOGEXT;'
     nnodes: 4
-    ppn: 12
+    ppn: 1
+    partition: bigmem
```

Comment:
@natalie-perlin One follow-up question: I hadn't thought about the implications of this change before now, is this going to work on all platforms? Shouldn't this be included in a machine file instead? I'm wondering if this is the source of the failures that @MichaelLueken is seeing.

Reply:
@mkavulich @MichaelLueken - 
bigmem is another issue that was required for reading native-level, thank you for pointing this out! It need to roll back to the previously use workflow parameters in `parm/wflow/coldstart.yaml` for _task_make_ics_mem#mem#_ and _task_make_lbcs_mem#mem#_ . Changing it now.

Reply:
@mkavulich - SRW is not currently moving ahead with implementing changes to read large grib2 files from RRFSv1 native-level forecasts. It would require another couple of PRs into the authoritative UFS_UTILS repo. So no changes needed at this point.